desks-web-app/src/components/views/BookingsView.vue

```diff
@@ -81,6 +81,17 @@
 
 </template>
 
+<style scoped>
+  #todayCard v-card {
+    height: 100%;
```

Comment:
I guess your intention was that both cards have the same height it seems not to work as expected.

![desks-different-card-hights_2](https://user-images.githubusercontent.com/1823283/196420363-33a95887-8878-47e6-abe1-47f340da4138.PNG)
![desks-different-card-hights](https://user-images.githubusercontent.com/1823283/196420371-82ea34b2-0052-4adf-bd54-eb8ddf8a23f0.PNG)

Reply:
oh well, right. should have added it to both cards. gonna fix it.